modules/face/include/opencv2/face/facemark.hpp

```diff
@@ -0,0 +1,111 @@
+#ifndef __OPENCV_FACEMARK_HPP__
+#define __OPENCV_FACEMARK_HPP__
+
+#include <iostream>
+#include <vector>
+#include <string>
+
+#include "opencv2/objdetect.hpp"
+#include "opencv2/objdetect/objdetect_c.h"
+#include "opencv2/imgproc/types_c.h"
+namespace cv {
+namespace face {
+    CV_EXPORTS_W bool getFacesHAAR( InputArray image,
```

Comment:
To add these functions to the documentation you have to wrap this block with doxygen instructions:
```.cpp
namespace cv { namespace face {
//! @addtogroup face
//! @{
...
//! @}
}}
```